cmd/api/routers/routers.go

```diff
@@ -18,13 +18,32 @@ type Controller struct {
 func (c *Controller) GetAllResources(context *gin.Context) {
 	group := context.Param("group")
 	version := context.Param("version")
-	// FIXME - Title is deprecated and this conversion must be done in a different way
-	resource := strings.Title(context.Param("resourceType")) //nolint:staticcheck
-	resources, err := c.Database.QueryResources(context, resource[:len(resource)-1], group, version)
+	resource := getResourceKindFromPlural(context.Param("resourceType"))
+	resources, err := c.Database.QueryResources(context, resource, group, version)
 
 	if err != nil {
 		context.JSON(http.StatusInternalServerError, err.Error())
 	}
 
 	context.JSON(http.StatusOK, resources)
 }
+
+func (c *Controller) GetNamespacedResources(context *gin.Context) {
+	group := context.Param("group")
+	version := context.Param("version")
+	namespace := context.Param("namespace")
+	resource := getResourceKindFromPlural(context.Param("resourceType")) + "/" + context.Param("namespace")
+	resources, err := c.Database.QueryNamespacedResources(context, resource, group, version, namespace)
+
+	if err != nil {
+		context.JSON(http.StatusInternalServerError, err.Error())
+	}
+
+	context.JSON(http.StatusOK, resources)
+}
+
+func getResourceKindFromPlural(plural string) string {
+	// FIXME - Title is deprecated and this conversion must bbe done in a different way
+	capitalized := strings.Title(plural) //nolint:staticcheck
+	return capitalized[:len(plural)-1]
```

Comment:
Are you just chopping of the last character here for now?

And can this be done all in one line?  Is there any advantage to creating `capitalized`?  Could it just become this?
```
return strings.Title(plural)[:len(plural)-1]
```

Reply:
I've chosen to do in two steps because of the `//nolint:staticcheck` comment, IIUC, that way the linter is just disabled in the `Title` operation and not in the other one.